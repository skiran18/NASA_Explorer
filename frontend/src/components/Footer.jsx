import React from "react";

const Footer = () => {
  return (
    <footer>
      <p>&copy; 2025 NASA Explorer</p>
    </footer>
  );
};

export default Footer;