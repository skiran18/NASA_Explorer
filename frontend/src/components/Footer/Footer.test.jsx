import React from "react";
import { render, screen } from "@testing-library/react";
import Footer from "./Footer";
import "@testing-library/jest-dom";

describe("Footer Component", () => {
  test("renders the footer with correct text", () => {
    render(<Footer />);
    const footerText = screen.getByText(/Â© 2025 NASA Explorer/i);
    expect(footerText).toBeInTheDocument();
  });

  test("matches snapshot", () => {
    const { container } = render(<Footer />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
