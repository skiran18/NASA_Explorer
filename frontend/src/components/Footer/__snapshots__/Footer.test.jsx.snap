// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component matches snapshot 1`] = `
<footer>
  <p>
    Â© 2025 NASA Explorer
  </p>
</footer>
`;
